src/LaunchDarkly.ServerSdk.DynamoDB/Integrations/DynamoDB.cs

```diff
@@ -0,0 +1,45 @@
+﻿
+namespace LaunchDarkly.Client.Integrations
+{
+    /// <summary>
+    /// Integration between the LaunchDarkly SDK and DynamoDB.
+    /// </summary>
+    public static class DynamoDB
+    {
+        /// <summary>
+        /// Name of the partition key that the data store's table must have. You must specify
+        /// this when you create the table. The key type must be String.
+        /// </summary>
+        public const string DataStorePartitionKey = "namespace";
+
+        /// <summary>
+        /// Name of the sort key that the data store's table must have. You must specify this
+        /// when you create the table. The key type must be String.
+        /// </summary>
+        public const string DataStoreSortKey = "key";
+
+        /// <summary>
+        /// Returns a builder object for creating a Redis-backed data store.
+        /// </summary>
+        /// <remarks>
+        /// This object can be modified with <see cref="DynamoDBDataStoreBuilder"/> methods for any desired
+        /// custom Redis options. Then, pass it to <see cref="Components.PersistentDataStore(Interfaces.IPersistentDataStoreAsyncFactory)"/>
```

Comment:
A couple references to Redis here that should probably be updated.

Reply:
Argh, copypasta strikes again - I could've sworn I did a global search for "Redis"...